common/eqemu_logsys.h

```diff
@@ -143,6 +143,13 @@ namespace Logs {
 		Corpses,
 		XTargets,
 		EvolveItem,
+		BotSettings,
+		BotPreChecks,
+		BotHoldChecks,
+		BotDelayChecks,
+		BotThresholdChecks,
+		BotSpellTypeChecks,
+		TestDebug,
```

Comment:
We already have a "Debug" category which is discouraged to be used for a more specific logging category

Reply:
I debated this for a bit but figured I'd throw a shot in the dark at it staying. I can remove if you want, I only prefer having it for easy debugging without having to do full rebuilds adding a new logging category as Debug itself has quite a few things attached to it automatically.

Reply:
My point is "Debug" is an "Misc" category that isn't useful to lump things into, we shouldn't have it. "Test Debug" is only inflating the issue. We shouldn't have either and a log likely has a better category to fall under.